sandbox/src/helpers/getUrlParameter.js

```diff
@@ -0,0 +1,8 @@
+export default (name) => {
+  name = name.replace(/[[]/, "\\[").replace(/[\]]/, "\\]");
```

Comment:
This could be greatly simplified with `URLSearchParam`.

Example:

```

/**
 * @param {string} name
 * @returns {string | null}
 */
export default (name) => {
  const params = new URLSearchParams(window.location.search)
  return params.get(name);
}
```